test/test_vcard_wrapper.py

```diff
@@ -464,11 +464,16 @@ def test_setting_and_getting_webpage_ablabel(self):
         wrapper._add_webpage({'github': 'https://github.com/scheibler/khard'})
         wrapper._add_webpage('http://example.com')
         self.assertListEqual(wrapper.webpages, [
-            'github: https://github.com/scheibler/khard',
+            {'github': 'https://github.com/scheibler/khard'},
             'http://example.com'])
 
-    @unittest.expectedFailure
     def test_labels_on_structured_values(self):
         vcard = VCardWrapper(_from_file('test/fixture/vcards/labels.vcf'))
-        # TODO find a solution for issue #221
-        self.assertListEqual(vcard.organisations, ['Test Inc'])
+        self.assertListEqual(vcard.organisations, [{'Work': ['Test Inc']}])
+
+    def test_setting_and_getting_webpages(self):
+        vcard = create_test_vcard()
+        wrapper = VCardWrapper(vcard)
+        wrapper._delete_vcard_object("FN")
```

Comment:
please make at least two tests for this: one to test setting and getting of some labeled attribute and one for reconstructing the fn from a labeled attribute.

Reply:
I just want to check what the new tests are testing for.

The other tests in this block test basic ABLABELs with webpages and loading the labelled `ORG` from the test file. This one ensures that the FN reconstruction works from a labelled `ORG`.

There's also a test in `test_yaml.py` for loading labelled/unlabelled webpages from yaml input.